.github/workflows/python.yml

```
@@ -0,0 +1,90 @@
+name: webviz-subsurface-components
+
+defaults:
+  run:
+    working-directory: ./python
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+      - published
+  schedule:
+    # Run CI daily and check that tests are working with latest dependencies
+    - cron: "0 0 * * *"
+
+jobs:
+  python:
+    # Run on all events defined above, except pushes which are not to master
+    if: github.event_name != 'push' || github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - python-version: "3.8"
+          - python-version: "3.9"
+          - python-version: "3.10"
```

Can be simplified to
```suggestion
      matrix:
        python-version: ["3.8", "3.9", "3.19"]
```